packages/api/src/schema/schema.graphql

```diff
@@ -162,6 +168,8 @@ type Hook {
   bitbucketUsername: String
   bitbucketToken: String
   bitbucketBuildName: String
+  slackEventFilter: String
+  slackBranchFilter: [String]
```

Comment:
That's very unfortunate duplication :( Would be nice to eliminate it eventually

Reply:
Agree, but seems as out of scope of the change.